scripts/release-build.test.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+const assert = require('assert');
+const { execSync } = require('child_process');
+
+const prebuiltPath = '../build';
+
+describe('release-build', () => {
+  it('pre-built bundles are identical to the source', () => {
+    execSync('npm run release-build');
+    const regular = require('..');
+    const bundled = require(prebuiltPath);
+    assert.deepEqual(regular, bundled);
+  }).timeout(5000); // Timeout must be long enough for all the file I/O
```

Comment:
Is the timeout needed, or does it only affect the color of the test in the output now?

Reply:
Yes, if a test exceeds the limit (in this case 2s), Mocha will fail it.

Reply:
I think so.
If default timeout is exceeded (2s), then the following message will be printed:
>   1) release-build
>       pre-built bundles are identical to the source:
>     Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()"
 is called; if returning a Promise, ensure it resolves. (C:\browser-compat-data\
scripts\release-build.test.js)
>      at processImmediate (node:internal/timers:464:21)

On "Maintenance LTS" (Node 10) configuration this test can easily exceed the limit. See this run from the most recent build on this branch:
https://github.com/mdn/browser-compat-data/runs/2351499801

Reply:
Hmm, I don't really understand what it could mean for a sync test to timeout, but it doesn't matter, this works :)

Reply:
I think it's Mocha that's timing out, not the test. It's a bit poorly documented, but my hunch here is that Mocha is trying to fail on slow tests (for example, to fail on infinite loops), rather than running endlessly.